lib/ansible/modules/database/mongodb/mongodb_replicaset.py

```diff
@@ -10,178 +10,145 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: mongodb_replicaset
 short_description: Initialises a MongoDB replicaset before authentication has been turned on and then validates the configuration when it has been turned on.
```

Comment:
This should be a lot shorter, although I didn't know how to summarize best.

Reply:
"Initialises a MongoDB replicaset" will suffice.